### PR TITLE
Cleanup/use power iterations

### PR DESCRIPTION
**Pull request by cpviolator**
Minor PR to ensure that the test files exhibit the default behaviour of using power iterations to guess the chebyshev polynomial maximum.

### CODE REVIEW

File: tests/eigensolve_test.cpp [hunk 1]
```diff
@@ -64,7 +64,10 @@ void display_test_info()
   if (eig_use_poly_acc) {
     printfQuda(" - Chebyshev polynomial degree %d\n", eig_poly_deg);
     printfQuda(" - Chebyshev polynomial minumum %e\n", eig_amin);
-    printfQuda(" - Chebyshev polynomial maximum %e\n\n", eig_amax);
+    if (eig_amax < 0)
```
Comment (maddyscientist) [inline tests/eigensolve_test.cpp]: Shouldn't this be `<= 0.0` ?

File: tests/multigrid_evolve_test.cpp [hunk 2]
```diff
@@ -68,7 +68,10 @@ display_test_info()
       if (mg_eig_use_poly_acc[i]) {
         printfQuda(" - level %d Chebyshev polynomial degree %d\n", i + 1, mg_eig_poly_deg[i]);
         printfQuda(" - level %d Chebyshev polynomial minumum %e\n", i + 1, mg_eig_amin[i]);
-        printfQuda(" - level %d Chebyshev polynomial maximum %e\n", i + 1, mg_eig_amax[i]);
+        if (mg_eig_amax[i] < 0)
```
Comment (maddyscientist) [inline tests/multigrid_evolve_test.cpp]: Again: `<=` ?

File: tests/staggered_multigrid_invert_test.cpp [hunk 3]
```diff
@@ -94,7 +94,10 @@ void display_test_info()
       if (mg_eig_use_poly_acc[i]) {
         printfQuda(" - level %d Chebyshev polynomial degree %d\n", i + 1, mg_eig_poly_deg[i]);
         printfQuda(" - level %d Chebyshev polynomial minumum %e\n", i + 1, mg_eig_amin[i]);
-        printfQuda(" - level %d Chebyshev polynomial maximum %e\n", i + 1, mg_eig_amax[i]);
+        if (mg_eig_amax[i] < 0)
```
Comment (maddyscientist) [inline tests/staggered_multigrid_invert_test.cpp]: And here 😉

File: tests/eigensolve_test.cpp (hunk 1 above)
Comment (cpviolator) [reply tests/eigensolve_test.cpp]: Yes...

File: tests/multigrid_evolve_test.cpp (hunk 2 above)
Comment (cpviolator) [reply tests/multigrid_evolve_test.cpp]: ... and yes.

**Pull request merged**